backend/src/core/utils/attachment.ts

```diff
@@ -1,6 +1,15 @@
+import crypto from 'crypto'
 import { UploadedFile } from 'express-fileupload'
 
-const ensureAttachmentsFieldIsArray = (
+export interface Attachment {
```

Comment:
Note: find type exported by library if available

Reply:
actually, I just went in and check, `UploadedFile` is the correct file type, can delete this `Attachment` interface